addons/purchase_stock/models/stock.py

```diff
@@ -257,6 +257,27 @@ def _set_default_route_id(self):
             orderpoint_wh_supplier.route_id = route_id[0].id
         return super()._set_default_route_id()
 
+    def _get_min_qty_to_order(self):
+        """ Calculates the minimum quantity that can be ordered according to the PO UoM
+        """
+        self.ensure_one()
+        min_qty_order_possible = super()._get_min_qty_to_order()
+        if min_qty_order_possible:
+            return min_qty_order_possible
+        elif self.route_id:
+            rules_actions = self.route_id.rule_ids.mapped('action')
+            if "buy" in rules_actions:
+                order_uom = self.product_id.uom_po_id
+                seller_ids = self.product_id.product_tmpl_id.mapped('seller_ids')
```

Comment:
IMO, You should _select_seller ?

Reply:
Thanks, I also didn't know this one :-)

Reply:
Actually the problem I see with `_select_seller` is that I also want to get sellers that have a min_qty higher than the quantity I need. I would need to pass an arbitrary high value for the `quantity` parameter to be able to mimic the previous behavior, which is not the prettiest thing.

Reply:
Alright, I understand, I didn't take that in account.